cajita/src/Cajita_LocalMesh.hpp

```diff
@@ -267,12 +284,16 @@ class LocalMesh<Device, NonUniformMesh<Scalar, NumSpaceDim>>
         // Compute the ghosted low corner
         for ( std::size_t d = 0; d < num_space_dim; ++d )
         {
+            // Interior domain case.
```

Comment:
`onLowBoundary`?

Reply:
Good point - we could just use that boolean here.